### PR TITLE
feat: add game mode classic 10 spell

### LINKED ISSUE #5: implement game mode: classic

### PR DESCRIPTION
**Pull request by RiskyFeryansyahP**
## TODO
- [X] Add classic mode 10 spell
- [X] Show record in seconds when point already 10 (10 spell)
- [x] Improve code logic

### CODE REVIEW

File: cmd/classic/main.go [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+			m.invoked[1] = m.invoked[0]
+			m.invoked[0] = i
+
+			if i == m.spell {
+				m.spell = generate(i)
+				m.point++
+			}
+
+			if m.point == 10 {
+				m.record = time.Since(m.timeSince).Seconds()
```
Comment (ronaudinho) [inline cmd/classic/main.go:116]: `return m, tea.Quit` after?

File: cmd/classic/main.go [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+		s += " "
+		s += spellMap[i]
+		s += " |"
+	}
+	s += "\n--------------------------------------------------\n"
+
+	s += fmt.Sprintf("%d POINTS\n\n", m.point)
+
+	if m.point == 10 {
+		s += "Your injoker classic Record is:\n"
```
Comment (ronaudinho) [inline cmd/classic/main.go]: injoker xD maybe capitalize all for uniformity?

File: cmd/classic/main.go [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+	}
+
+	return spellValue[invokerOrb]
+}
+
+func generate(prevSpell spell) spell {
+	randSource := rand.NewSource(time.Now().Unix())
+	r := rand.New(randSource)
+
+	randSpellNumber := r.Intn(10-1) + 1
```
Comment (ronaudinho) [inline cmd/classic/main.go:185]: `r.Intn(9) + 1`, well, unless this is a new DDoS technique I didnt know about

File: cmd/classic/main.go (hunk 1 above)
Comment (RiskyFeryansyahP) [reply cmd/classic/main.go:116]: Ahhhh, sorry, I forgot to move [this part](https://github.com/ronaudinho/inv/blob/787200027e58acd994e060302cf144112a067b16/cmd/classic/main.go#L94-L96) to the line you mentioned above. It should return after the point reaches 10 (10 spells equals 10 points).

File: cmd/classic/main.go (hunk 2 above)
Comment (RiskyFeryansyahP) [reply cmd/classic/main.go]: Only `Injokers` play this game xD, Real Invokers play in ranked matches and don't need training 🤣

Okaay, i will change it

File: cmd/classic/main.go (hunk 1 above)
Comment (RiskyFeryansyahP) [reply cmd/classic/main.go:116]: updated

File: cmd/classic/main.go (hunk 2 above)
Comment (RiskyFeryansyahP) [reply cmd/classic/main.go]: updated

File: cmd/classic/main.go (hunk 3 above)
Comment (RiskyFeryansyahP) [reply cmd/classic/main.go:185]: Yup, it works if u want to generate random number using min max value

File: cmd/classic/main.go (hunk 2 above)
Comment (ronaudinho) [reply cmd/classic/main.go]: real invokers play WE and buy right clicks.

**Review by ronaudinho — APPROVED** [review]
latm

**Pull request merged**